DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesVfr.vfr

```diff
@@ -233,6 +233,530 @@ formset
              option text = STRING_TOKEN(STR_ME_MODE_DISABLE_HAP), value = ME_MODE_DISABLE_HAP, flags = 0;
           endif;
       endoneof;
+      grayoutif TRUE;

```

Comment:
Add an `disableif` for PCD that will control the visibility of this status information, above the `grayoutif`.

Reply:
https://github.com/Dasharo/edk2/pull/248/commits/c82ebb6d73b8275b12baf834bfca5c1e237c3903